cumulus_library/template_sql/insert_into.sql.jinja

```diff
@@ -12,6 +12,8 @@ VALUES
     {%- for field in row -%}
     {%- if table_cols[loop.index0] in type_casts.keys() -%}
     {{ type_casts[table_cols[loop.index0]] }} '{{ field }}'
+    {%- elif not field -%}
+    NULL
```

Comment:
not actually a bug per se, just the first time we actually tried to explicitly insert null data via this template.

Reply:
I've hit errors with Athena where it likes that kind of thing as `CAST(NULL AS VARCHAR)` - are you gonna need that here too?

Reply:
almost certaintly, yeah